### PR TITLE
Added API to use JH database

### PR DESCRIPTION
**Pull request by mahyar-osn**
This PR is partly related to issue (https://github.com/ABI-Covid-19/moh-data/issues/9). 

- We can now fetch the recovered and dead cases (cumulative) from JH database and combine it with the MoH data that we were already getting.
- Also, the code now automatically fills in the missing dates in the MoH data. Thanks to @agarny for detecting this issue.

### CODE REVIEW

File: src/moh_data/main.py [hunk 1]
```diff
@@ -230,6 +250,8 @@ def export_data(data, filename):
         print('Cumulative confirmed cases on day 41 = ', run_data.get_cumulative_confirmed_cases_on_day(40))
         print('Cumulative probable cases on day 41 = ', run_data.get_cumulative_probable_cases_on_day(40))
         print('Cumulative total cases on day 41 = ', run_data.get_cumulative_total_cases_on_day(40))
```
Comment (agarny) [inline src/moh_data/main.py]: These should actually refer to day 40.

File: src/moh_data/main.py [hunk 2]
```diff
@@ -230,6 +250,8 @@ def export_data(data, filename):
...
+        print('Cumulative recovered cases on day 80 = ', run_data.get_cumulative_recovered_cases_on(80))
+        print('Cumulative dead cases on day 80 = ', run_data.get_cumulative_dead_cases_on(80))
```
Comment (agarny) [inline src/moh_data/main.py]: The day reference is wrong. I mean that day 80 for JHU corresponds to day 43 for NZ (day 0 is 28 February, I think).

**Review by agarny — CHANGES_REQUESTED** [review]
Looks good pending one minor issue and one "major" issue.

**Review by agarny — APPROVED** [review]

**Comment by mahyar-osn** [comment]
Thanks Alan for fixing the issues and approving the PR!

**Pull request merged**